airflow/api_connexion/security.py

```diff
@@ -0,0 +1,37 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from functools import wraps
+from typing import Callable, TypeVar, cast
+
+from flask import Response, current_app
+
+from airflow.api_connexion.exceptions import Unauthenticated
+
+T = TypeVar("T", bound=Callable)  # pylint: disable=invalid-name
```

Comment:
Is this disabled necessary? If yes, maybe we can add this `T` to accepted names as we are using it quite often?

Reply:
Any letter of the alphabet can be used as a variable name, so I don't think that's a good idea. We don't have a lot of such situations in the code. You might get this feeling because I started using them more, but overall we have less than 10 such situations in the codebase.